.github/workflows/issue-cleanup.yml

```diff
@@ -35,12 +36,32 @@ jobs:
             }
 
             const body = lines.join('\n')
+            
+            const labels = issue.data.labels.map(label => label.name)
+            
+            if (products !== '') {
+              switch (true) {
+                case products.include('Data Grid'):
+                  labels.push('component: data grid')
+                  break
```

Comment:
Is there a reason to use switch here? 🤔

Reply:
not super attached to the `switch` statement. I just felt this is more readable than multiple `if`s

thinking about this: It can be that there are multiple components selected, so it might be worthwhile to do something like this:
``tsx
const productMap = {
  'Data Grid': 'data grid',
  'Date and Time Pickers': 'pickers',
  'Charts': 'charts',
  'Tree View': 'tree view',
}

if (products !== '') {
  products.split(',').forEach(product => {
    labels.push(productMap[product])
  })
}
```

WDYT?

Reply:
Ah, if it is possible to have multiple selected then the later would be better indeed. 

I thought you were using `switch` for some kind of optimisation 😆

Reply:
changed 👍🏼